packages/check-time/extra/check-time.c

```diff
@@ -0,0 +1,41 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <sys/timex.h>
+#include <string.h>
+#include <errno.h>
+
+#define USEC_PER_MSEC 1000L
+#define MAX_EST_ERROR_US 100L * USEC_PER_MSEC /* 100 millisecond */
+
+int main() {
+	struct timex tx;
```

Comment:
tx {};
rc = 0;
error = 0;

to prevent fun default not initializing (the curly braces cause the C compiler to fill all not explicitly set fields with zeros).